spec/models/user_spec.rb

```diff
@@ -182,30 +182,30 @@ def within_duration_window_days
     end
   end
 
-  describe 'blacklist' do
+  describe 'email domains denylist integration' do
     around do |example|
-      old_blacklist = Rails.configuration.x.email_blacklist
```

Comment:
I think the previous config value here never actually had anything in it, so this may technically be a bugfix and not just a rename, although it's one that I think doesn't really matter in the spec scenario.